install_kafka.yml

```diff
@@ -0,0 +1,92 @@
+---
+  - name: "Install Kafka"
+    hosts: kafkaserver1
+
+
+    roles:
+    - install-JDK-role
+
+    tasks:
+
+      #  - name: Install the latest version of JDK
+      #    become: yes
+      #    yum:
+      #     name: jdk-11
+      #     state: latest
+
+    
+       - name: Copy Kafka Binaries
+         become: yes 
+         become_user: dojokafka
+         copy:
+           src: /Users/rafiahmadjan/Documents/DOJO/Dojo-Mini-Kafka/files/kafka_2.13-3.2.0.tgz
+           dest: '{{ home }}'
+           owner: dojokafka
+           group: dojo
+           mode: 0644
+         
+
+       - name:  untar the binaries
+         become: yes 
+         become_user: dojokafka
+         ansible.builtin.unarchive:
+           src: "{{ home }}/kafka_2.13-3.2.0.tgz"
```

Comment:
f.eks. `kafka_2.13-3.2.0.tgz` can reside inside vars for it is uses across the playbook

Reply:
kafka_2.13-3.2.0.tgz  is in a variable now called from variables.yml